### DISCUSSION TITLE
Absolute  qualified names `.List.foldLeft`

### DISCUSSION THREAD

**Discussion by JonathanStarup** [flix/flix#8492] [Ideas]
If i write this
```
def myFunction(): Random = ...
```
What `Random` am I speaking about, specifically? It can be `Random` or `Current.Namespace.Random`.

This is annoying if, e.g., I am implementing `eff Random` and want to have a handler using the existing `Random`

## Proposal
Allow prefix `.` when specifying names, that means that I can write `handler(f: a -> b \ ef, r: .Random): (a -> b \ ef - Random) \ IO = ...`. The feature is that `.Random` always refer to the root random, not the current random.

We don't use `.` much in Flix so I don't think parsing would be an issue

**Comment by mlutze** [comment]
IMO Standard library just needs to be in the Flix namespace. (Same with packages -- they should be qualified with something)

**Comment by JonathanStarup** [comment]
I like that approach. It is however a bit taxing since you cant do `use flix._` and we don't have editor support for uses :L

**Comment by mlutze** [comment]
> I like that approach. It is however a bit taxing since you cant do `use flix._` and we don't have editor support for uses :L

Sounds like we should get editor support for uses :D
